Gemfile.lock

```diff
@@ -1,94 +1,94 @@
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     aws-eventstream (1.1.0)
-    aws-partitions (1.353.0)
-    aws-sdk-core (3.104.3)
+    aws-partitions (1.399.0)
+    aws-sdk-core (3.109.3)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.36.0)
-      aws-sdk-core (~> 3, >= 3.99.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.76.0)
-      aws-sdk-core (~> 3, >= 3.104.1)
+    aws-sdk-s3 (1.85.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.1)
+    aws-sigv4 (1.2.2)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.15)
-    better_errors (2.7.1)
+    bcrypt (3.1.16)
+    better_errors (2.9.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
     bindex (0.8.1)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.7)
+    bootsnap (1.5.1)
       msgpack (~> 1.0)
     builder (3.2.4)
     byebug (11.1.3)
     coderay (1.1.3)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     debug_inspector (0.0.3)
-    devise (4.7.2)
+    devise (4.7.3)
```

Comment:
### [devise](https://github.com/heartcombo/devise)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.7.2 → 4.7.3

[_~~change-log~~, [source-code](http://github.com/plataformatec/devise)_]





<details>
<summary>Commits</summary>

A change of **28** commits. See the full changes on [the compare page](https://github.com/heartcombo/devise/compare/v4.7.2...v4.7.3).

These are the first 10 commits:
- (6991ff4) [Oops, fix changelog [ci skip]](https://github.com/heartcombo/devise/commit/6991ff4eb42bb8a197f8c83ed9bcf46d50392083)
- (0e33f55) [Fixes broken image (#5253) [ci skip]](https://github.com/heartcombo/devise/commit/0e33f55b7d6fea66d0c73ebc73a597a16819a450)
- (b25492e) [Remove Heroku anchor link and fix some typos](https://github.com/heartcombo/devise/commit/b25492ea0185bd1525e65f4c56d465b79ceff689)
- (79d7edd) [Merge pull request #5256 from ptcodes/remove_heroku_anchor_l…](https://github.com/heartcombo/devise/commit/79d7eddfc30eef4c7d1803c2623fa61fb74f156f)
- (4f60544) [Improve deprecation message with example of how to remove it](https://github.com/heartcombo/devise/commit/4f60544396824c516bdb5c165f9631d727b9560d)
- (ffa8a80) [Fix warning from `thor`](https://github.com/heartcombo/devise/commit/ffa8a80f423ee1cad4b19bb61e8e3977edc92e5a)
- (c249ba9) [Merge pull request #5258 from deivid-rodriguez/fix_thor_warn…](https://github.com/heartcombo/devise/commit/c249ba991b98634cc2ffee2def5a3683d1e6a7ae)
- (057afdc) [Fix another thor deprecation warning in the install generato…](https://github.com/heartcombo/devise/commit/057afdc1e6cadcb4102521edb604225da897965a)
- (b94b957) [Prefer american style english for code](https://github.com/heartcombo/devise/commit/b94b957490359a228ed5d4c76b8c1af69b290737)
- (f5cc775) [Remove commented out code](https://github.com/heartcombo/devise/commit/f5cc775a5feea51355036175994edbcb5e6af13c)


</details>